src/api/app/controllers/webui/users/tasks_controller.rb

```diff
@@ -1,7 +1,11 @@
 module Webui
   module Users
     class TasksController < WebuiController
-      before_action :require_login
+      # TODO: Remove this when we'll refactor kerberos_auth
+      before_action :kerberos_auth
+      before_action -> { authorize([:users, :task]) }
```

Comment:
I've opted to use an [headless policy](https://github.com/varvet/pundit#headless-policies) because I didn't find any model to associate.